pivot-growth.jl

```diff
@@ -146,10 +151,51 @@ function run_model(n)
     val,B,MOI.FEASIBLE_POINT
 end
 
+"""
+    run_model_new(n)
+
+Maximize pivot growth for real matrix with complete pivoting.
+"""
+function run_model_new(n)
+    start_matrix = randn(n, n)
+    thestart = genp(geperm(start_matrix))
+    thestart = genp(thestart[:,:,1] * Diagonal(sign.(thestart[k,k,k] for k = 1:n)))
+    thestart ./= thestart[1,1,1]
+    model = Model(Ipopt.Optimizer)
+    set_optimizer_attribute(model, "max_iter", 500)
+    set_silent(model)
+    indices = [(i, j, k) for k in 1:n for i in k:n for j in k:n]
+    @variable(model, x[(i,j,k) in indices], start = thestart[i,j,k])
```

Comment:
Are there any reasonable bounds on all of `x`? For example, what is a reasonable bound for `x[1, 1, 2]`?

Reply:
@odow  Thanks oscar.  Can you tell us why that might be helpful?  i mean it could be 5k where k is the third index , but how would that help?

Reply:
In general, having variable bounds is better than not, and having tight variable bounds is even better. It can help with the line search where the solver doesn't wander into extreme parts of the variable domain.

But I think these are smaller issues. The big problem is that the model is non-convex.

Reply:
What's the goal for input dimension?